src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -81,6 +81,104 @@ def to_tuple(self) -> Tuple[Any]:
         )
 
 
+@dataclass
+# Copied from transformers.models.clip.modeling_clip.CLIPOutput with CLIP->Blip2ImageTextMatchingModel
+class Blip2ImageTextMatchingModelOutput(ModelOutput):
+    """
+    Args:
+        loss (`torch.FloatTensor` of shape `(1,)`, *optional*, returned when `return_loss` is `True`):
+            Contrastive loss for image-text similarity.
+        logits_per_image (`torch.FloatTensor` of shape `(image_batch_size, text_batch_size)`):
+            The scaled dot product scores between `image_embeds` and `text_embeds`. This represents the image-text
+            similarity scores.
+        logits_per_text (`torch.FloatTensor` of shape `(text_batch_size, image_batch_size)`):
+            The scaled dot product scores between `text_embeds` and `image_embeds`. This represents the text-image
+            similarity scores.
+        text_embeds (`torch.FloatTensor` of shape `(batch_size, output_dim`):
+            The text embeddings obtained by applying the projection layer to the pooled output of [`Blip2ImageTextMatchingModelTextModel`].
+        image_embeds (`torch.FloatTensor` of shape `(batch_size, output_dim`):
+            The image embeddings obtained by applying the projection layer to the pooled output of [`Blip2ImageTextMatchingModelVisionModel`].
+        text_model_output (`BaseModelOutputWithPooling`):
+            The output of the [`Blip2ImageTextMatchingModelTextModel`].
+        vision_model_output (`BaseModelOutputWithPooling`):
+            The output of the [`Blip2ImageTextMatchingModelVisionModel`].
```

Comment:
Don't think the `Blip2ImageTextMatchingModelTextModel` class exists

Reply:
Thanks, docstring was updated.